src/WebJobs.Script/Binding/EasyTableBinding.cs

```diff
@@ -22,6 +22,9 @@ public class EasyTableBinding : FunctionBinding
         {
             TableName = metadata.TableName;
             Id = metadata.Id;
+            MobileAppUri = metadata.Connection;
```

Comment:
Yeah, I think my naming worked out :) "Connection" does make sense for a URI to connect to as well :)